### PR TITLE
chore: migrate off of css-selector-tokenizer

### PR DESCRIPTION
**Pull request by jquense**
This is pretty hard :/ And i don't think there is way to be super confident that there isn't some subtle change in behavior due to api differences, unclear behavior, limited tests.

I think actually the right thing to do here is REMOVE support for the non-nested selector. the open ended selector is weird and inconsistent, even in the logic of this plugin there is unclear transformations. and I even added a test to ensure i wasn't changing something but the old behavior that is definitely wrong.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,9 +18,15 @@
     "type": "git",
     "url": "https://github.com/css-modules/postcss-modules-local-by-default.git"
   },
+  "prettier": {
+    "semi": true,
+    "singleQuote": true,
+    "trailingComma": "es5"
+  },
   "dependencies": {
     "css-selector-tokenizer": "^0.7.0",
```
Comment (evilebottnawi) [inline package.json]: Remove this from deps and update lock file :+1:

**Comment by evilebottnawi** [comment]
I will continue development, merge this as is

**Comment by jquense** [comment]
thanks! sorry didn't have a moment to finish it up in the last two days!

**Comment by jquense** [comment]
yup, just the note you said

**Comment by evilebottnawi** [comment]
@jquense i will start testing new PR with css-loader

**Pull request merged**